### PR TITLE
[YUNIKORN-1289] Publish Daemonset scheduling design doc

### PR DESCRIPTION
**Pull request by manirajv06**

### CODE REVIEW

File: docs/design/simple_preemptor.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
+             ---
```
Comment (wilfred-s) [inline docs/design/simple_preemptor.md:1]: No spaces at the start of the line

File: sidebars.js [hunk 2]
```diff
@@ -75,6 +75,7 @@ module.exports = {
                     'design/state_aware_scheduling',
                     'design/scheduler_object_states',
                     'design/scheduler_core_design',
+                    'design/simple_preemptor'
```
Comment (wilfred-s) [inline sidebars.js:78]: This should not be in between the two old/no longer maintained docs. It should be higher up the list, around the generic resource line

**Review by wilfred-s — CHANGES_REQUESTED** [review]
two minor point to fix

**Review by craigcondit — APPROVED** [review]
+1 LGTM

**Pull request merged**